module/apmotel/gatherer_config.go

```diff
@@ -78,3 +86,23 @@ func customAggregationSelector(ik metric.InstrumentKind) aggregation.Aggregation
 		return metric.DefaultAggregationSelector(ik)
 	}
 }
+
+// WithTemporalitySelector configure the Aggregation Selector the exporter will
+// use. If no AggregationSelector is provided the DefaultAggregationSelector is
+// used.
+func WithTemporalitySelector(temporality metric.TemporalitySelector) GathererOption {
+	return func(cfg gathererConfig) gathererConfig {
+		cfg.temporality = temporality
+		return cfg
+	}
+}
+
+func defaultTemporalitySelector(ik metric.InstrumentKind) metricdata.Temporality {
+	switch ik {
+	case metric.InstrumentKindCounter, metric.InstrumentKindObservableCounter, metric.InstrumentKindHistogram:
+		return metricdata.DeltaTemporality
+	case metric.InstrumentKindUpDownCounter, metric.InstrumentKindObservableUpDownCounter, metric.InstrumentKindObservableGauge:
+		return metricdata.CumulativeTemporality
+	}
+	panic("unknown instrument kind")
```

Comment:
```suggestion
	}
	return metric.DefaultTemporalitySelector(ik)
```

I'd rather we didn't panic here, in case OTel grows new instrument kinds and we don't keep up to date.